server/portal/apps/workbench/urls.py

```diff
@@ -9,5 +9,6 @@
 urlpatterns = [
     re_path('account', IndexView.as_view(), name='account'),
     re_path('dashboard', IndexView.as_view(), name='dashboard'),
+    re_path('onboarding/admin', IndexView.as_view(), name='onboardingadmin'),
```

Comment:
Can we change the `name` to `onboarding_admin`? A bit more readable that way.
```suggestion
    re_path('onboarding/admin', IndexView.as_view(), name='onboarding_admin'),
```